src/downloader.rs

```diff
@@ -216,8 +216,15 @@ impl Downloader {
         };
 
         debug!("Creating destination file {:?}", &output);
+        // append: If we can't resume from where we left off,
+        //         we should overrwite the file and start again
+        //         This also prevents corrupting files by writing
+        //         to them again
+        // write:  We are writing to the file
+        // create: The file should be created if it doesn't exist
```

Comment:
Documentation is good to add in general, but in this case I don't think it is necessary to document each member of the builder. They are all well documented in the Tokio documentation.

Reply:
Oh right, sorry, I completely overlooked that `println`, and yeah that's fair
I'll fix both of those quick now